.gitignore

```diff
@@ -135,3 +135,5 @@ dist
 docs/
 apidocs/
 devdocs/
+
+.vercel

```

Comment:
This already exists in .gitignore, guessing was added by vercel automatically, just double check what files you're adding before commiting them, as this should be removed.

Reply:
Unresolving as this still exists

Reply:
I'll check on this with you in person tn since I'm kinda lost

Reply:
I think everything else is good to go 🫡🫡🫡